src/generator/AutoRest.Ruby.Azure/CodeGeneratorRba.cs

```diff
@@ -73,8 +72,11 @@ public CodeGeneratorRba()
                     continue;
                 }
 
-                var modelTemplate = new ModelTemplate { Model = model };
-                await Write(modelTemplate, Path.Combine(GeneratorSettingsRb.Instance.modelsPath, CodeNamer.UnderscoreCase(model.Name) + ImplementationFileExtension));
+                var modelTemplate = new AzureModelTemplate { Model = model };
```

Comment:
As discussed offline, since we have to create AzureModelTemplate anyway, it is good to keep the Azure specific model generation in one place than two. I am not convinced that we need to continue to use 2 file. No changes here